packages/web-react/src/components/SegmentedControl/demo/SegmentedControlWithTooltip.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { FillVariants } from '../../../constants';
+import { Icon } from '../../Icon';
+import { Tooltip, TooltipPopover, TooltipTrigger } from '../../Tooltip';
+import SegmentedControl from '../SegmentedControl';
+import SegmentedControlItem from '../SegmentedControlItem';
+
+const SegmentedControlWithTooltip = () => {
+  const [activeTooltipIndex, setActiveTooltipIndex] = useState<number | null>(null);
+  const [selectedValue, setSelectedValue] = useState<string | string[]>('value-1');
+
+  return (
+    <SegmentedControl
+      label="Label"
+      name="segmented-tooltip-icon"
+      selectedValue={selectedValue}
+      setSelectedValue={setSelectedValue}
+      variant={FillVariants.FILL}
+    >
+      {Array.from({ length: 5 }, (_, index) => {
+        const value = `value-${index + 1}`;
+        const id = `segmented-control-tooltip-js-${index + 1}`;
+        const isOpen = activeTooltipIndex === index;
+
+        return (
+          <Tooltip
+            key={value}
+            id={`tooltip-${value}`}
+            isOpen={isOpen}
+            onToggle={() => setActiveTooltipIndex(isOpen ? null : index)}
+          >
+            <TooltipTrigger
+              elementType={SegmentedControlItem}
+              value={value}
+              id={id}
+              aria-label={`Label Item ${index + 1}`}
+            >
+              <Icon name="file" boxSize={20} />
+              <TooltipPopover>Label</TooltipPopover>
+            </TooltipTrigger>
+          </Tooltip>
```

Comment:
```suggestion
          <Tooltip
            key={value}
            id={`tooltip-${value}`}
            isOpen={isOpen}
            onToggle={() => setActiveTooltipIndex(isOpen ? null : index)}
          >
            <TooltipTrigger
              elementType={SegmentedControlItem}
              value={value}
              id={id}
              aria-label={`Label Item ${index + 1}`}
            >
              <Icon name="file" boxSize={20} />
            </TooltipTrigger>
            <TooltipPopover>Label</TooltipPopover>
          </Tooltip>
```
😬 If I read the Tooltip docs correctly, the popover should be placed outside the trigger. I tested it locally, and imho, it works. Can you please update all demos and docs as well?

Reply:
updated